src/pages/Swap/LimitOrderBox/components/CurrencySelectTooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChainId, Currency, ETHER, XDAI } from '@swapr/sdk'
+
+import { MouseoverTooltip } from '../../../../components/Tooltip'
+import { useActiveWeb3React } from '../../../../hooks'
+import { wrappedCurrency } from '../../../../utils/wrappedCurrency'
+
+const wrappedCurrencySymbols: Record<Extract<ChainId, ChainId.MAINNET | ChainId.GNOSIS>, string | undefined> = {
+  [ChainId.MAINNET]: wrappedCurrency(ETHER, ChainId.MAINNET)?.symbol,
+  [ChainId.GNOSIS]: wrappedCurrency(XDAI, ChainId.GNOSIS)?.symbol,
+}
```

Comment:
can we try reusing some logic. There is useNativeCurrency hook and lots of other stuff that can be reused..

Reply:
Will see if we can reuse logic here

Reply:
Thanks man, this really helped to make a more generic solution.